### PR TITLE
Add CyberArk EPM workflow files

### PR DESCRIPTION
**Pull request by compostCY**
Hello team:
Jawine Westland from your team has QA the app within the last few months. She has posted this in the IBM page already. 
https://exchange.xforce.ibmcloud.com/hub/extension/6f4fd73ae721109d758d6807ba8e404c

She ask me to upload the xml files here by creating a fork and a pull request. I want to put this into the CyberArk EPM folder, but i can't seems to create the folder while uploading. Could you please ensure this is in the CyberArk EPM folder ? Thanks.
==
Use the "Add File" button to upload all your files. Note that to create a folder, you simply include a / in the file name to create (e.g. My Favorite Vendor/MyFavoriteVendor-Workflow.xml).
==
I was not able to create the folder using step above while uploading.

### CODE REVIEW

File: CyberArk-EPM-AggregatedPolicy-Workflow.xml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        <ClearStatus />
+		
+		<!-- Initialize doesn't overwrite with new value with one exists, Set overwrite with new value if one exists 
+			 use bookmark in PRODUCTION, but use set in Test with older data. change between *Initalize* and *Set* here.
+		-->
+		<!-- go back to how ever i want, bookmark seems to be stuck for some reason, need to figure out  why 7*24=168, 336 14 days -->
+		
+		<!-- Initialize the bookmark (Start date) to 1 days ago, if not set default is 1 day -->
+
+        <Initialize path="/epm_policy/bookmark" value="${time() - (60000 * 60 * 24 * 7)}" /> <!-- 1 days prior -->
```
Comment (ChrisCollinsIBM) [inline CyberArk-EPM-AggregatedPolicy-Workflow.xml:25]: This one is set to 7 days but in the comments it's indicated that it is 1 day.

**Review by ChrisCollinsIBM — CHANGES_REQUESTED** [review]
Thanks for the submission, I added a few comments around default values for parameters as well as some of the timestamps being used to initialize initial collection.

As well, we also need a README.md with a small overview of what the 3 workflows are for, and possibly where the users would configure the access and what level of access is required (standard permissions for example) to help ensure users don't just use an admin user with more permissions than required.

Thank you!

File: CyberArk-EPM-AdminAudits-Workflow.xml [hunk 2]
```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+CyberArk EPM AdminAudit API - QRadar Integration (Universal Cloud REST API)
+-->
+<Workflow name="CyberArkEPM" version="1.0" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V1">
+
+    <Parameters>
+	    <Parameter name="host" label="EPM Dispatcher URL" default="login.epm.cyberark.com" required="true" />
+        <Parameter name="username" label="Username" default="johndoe" required="true" />
```
[the remark this answers is not in the document]
Comment (compostCY) [reply CyberArk-EPM-AdminAudits-Workflow.xml:9]: Thanks Chris:
Got it, sure we can remove it. We can just make it required. As this is my first time coding this, i am following the patterns from others. I model this from from CyberArk Idaptive workflow where they have this as well:
     <Parameter name="username" label="Username" default="siemuser" required="true" />

I am ok removing it if you need to.

File: CyberArk-EPM-AggregatedEvents-Workflow.xml [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+		<!-- Clear status of log source -->
+        <ClearStatus />
+		
+		<!-- Initialize doesn't overwrite with new value with one exists, Set overwrite with new value if one exists 
+			 use bookmark in PRODUCTION, but use set in Test with older data. change between *Initalize* and *Set* here.
+		-->
+		
+		<!-- Initialize the bookmark (Start date) to 1 days ago, if not set default is 1 or  7 days or whatever customers want. -->
+
+        <Initialize path="/epm_event/bookmark" value="${time() - (60000 * 60 * 24 * 1)}" /> <!-- 7 days prior -->
```
[the remark this answers is not in the document]
Comment (compostCY) [reply CyberArk-EPM-AggregatedEvents-Workflow.xml:24]: my apology, this should default to 7 days out of the box.

File: CyberArk-EPM-AggregatedPolicy-Workflow.xml (hunk 1 above)
Comment (compostCY) [reply CyberArk-EPM-AggregatedPolicy-Workflow.xml:25]: apology, for policy event, it should default to 1 day, as there could be potential many policy record.

File: CyberArk-EPM-AdminAudits-Workflow.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ChrisCollinsIBM) [reply CyberArk-EPM-AdminAudits-Workflow.xml:9]: You would commit it as a file README.md into the same directory as the workflows.

File: CyberArk-EPM-AggregatedEvents-Workflow.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (ChrisCollinsIBM) [reply CyberArk-EPM-AggregatedEvents-Workflow.xml:24]: Can you update in the workflow please with a new commit? For this as well as the other one.

**Comment by ChrisCollinsIBM** [comment]
Hi @compostCY, the review issues are marked as resolved but I don't see another commit with the changes.

Can you please make the updates as well as add the README.md and then I'll get the PR merged in.

**Review by ChrisCollinsIBM — APPROVED** [review]
Thanks for the updates!

**Pull request merged**